server/jetty-app/README.md

```diff
@@ -1,70 +1,29 @@
 # server-jetty-app
 
-## Native development
-
 This document is oriented towards getting a server up for local development.
-If you are interested in running a native production release, please see
+If you are interested in running a native production release or to see other options for starting the server, please see
```

Comment:
No context is provided about what the difference in a dev env and a prod env are and what that means.